x-pack/legacy/plugins/maps/public/layers/sources/es_geo_grid_source/es_geo_grid_source.js

```diff
@@ -32,17 +32,21 @@ import { AbstractESAggSource } from '../es_agg_source';
 import { DynamicStyleProperty } from '../../styles/vector/properties/dynamic_style_property';
 import { StaticStyleProperty } from '../../styles/vector/properties/static_style_property';
 import { DataRequestAbortError } from '../../util/data_request';
+import { registerSource } from '../source_registry';
+import { registerLayerWizard } from '../../layer_wizard_registry';
 
 export const MAX_GEOTILE_LEVEL = 29;
 
+const clustersTitle = i18n.translate('xpack.maps.source.esGridClustersTitle', {
+  defaultMessage: 'Clusters',
```

Comment:
I wouldn;'t remove the reference to grids for continuity, since we've had that for almost a year now. Maybe "Grids and clusters", or just retain "Grid aggregation".